### ISSUE TITLE
Design: Restyle classifier dark mode buttons

### ISSUE THREAD

**Issue by seanmiller26** [zooniverse/front-end-monorepo#4413]
## Is your feature request related to a problem? Please describe.
We've had feedback from testing suggesting that the buttons do not stand out enough in dark mode.

## Describe the solution you'd like
Restyle the buttons to have more contrast against the background and better inform users when they are either active or inactive. 

<img width="1018" alt="Screen Shot 2023-03-31 at 11 28 56 AM" src="https://user-images.githubusercontent.com/89043489/229177496-c967e581-4742-4408-9677-c5b99769265f.png">

**Comment by seanmiller26** [comment]
This brings up a good point about how we use opactity on the inactive buttons. When inspecting it shows the font and border color as white, but it clearly is not.

<img width="422" alt="Screen Shot 2023-04-03 at 11 19 32 AM" src="https://user-images.githubusercontent.com/89043489/229568942-e252e8c5-c0a9-40c0-9360-724c90e7f351.png">

@mcbouslog how does this work, exactly?

**Comment by mcbouslog** [comment]
The task nav buttons are defined in the classifier library ([DoneAndTalk component link](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButton.js#L13-L19)) which use the react-components library [Primary Button](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-react-components/src/PrimaryButton/PrimaryButton.js). The Primary Button has custom themes per color (i.e. gold, blue, green, and teal - [link to files here](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-react-components/src/PrimaryButton/themes)) and I think if we want to change the disabled styling we'd do so in the color themes, like for [blue here](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-react-components/src/PrimaryButton/themes/blue.js#L32-L35).

Here's the related storybook - [Primary Button storybook](https://zooniverse.github.io/front-end-monorepo/@zooniverse/react-components/index.html?path=/story/components-primarybutton--default&args=color:blue;dark:true;disabled:true&globals=locale:en;locales.en:English;locales.test:Test+Language).

**Comment by seanmiller26** [comment]
A quick accessibility check finds that according to the WCAG color and contrast requirements:

> Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

And since we are already are using the disabled mouse icon on hover and a status label of 'disabled,' we are covering all of the other requirements.

**Comment by mcbouslog** [comment]
I didn't know such cases have no contrast requirements, that's interesting and good to know! Thanks and sounds good to me.
